### PR TITLE
Fixes, edits and tweaks

### LINKED ISSUE #101: Headings overflow off the page in mobile
image
Here the heading kattilapäivystykset expands off the page and makes the entire page side-scrollable.

### PR DESCRIPTION
**Pull request by lurppa**
Closes #101

### CODE REVIEW

**Comment by 45spoons** [comment]
This is getting quite big to review in one go, there seem to be a couple features in here that are not overlapping

**Comment by 45spoons** [comment]
Now its really getting out of hand

**Comment by lurppa** [comment]
Closes #99

File: layouts/shortcodes/column.html
[the remark this answers is not in the document]
Comment (45spoons) [reply layouts/shortcodes/column.html:1]: Related directly to this change: With the default value being an empty string, will this produce classes "col-md-" by default?

File: layouts/shortcodes/wrap.html
Comment (45spoons) [inline layouts/shortcodes/wrap.html:1]: For anyone wondering what this is, it's a hidden character that suggests a wordbreak with a hyphen.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr
https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens

File: layouts/shortcodes/column.html
[the remark this answers is not in the document]
Comment (45spoons) [reply layouts/shortcodes/column.html:1]: Looked at it closer, this isn't an issue 👍

**Review by ikaros02 — APPROVED** [review]

**Pull request merged**